### DISCUSSION TITLE
libraries for implimentig facial recognition in js ??

### DISCUSSION THREAD

**Discussion by omeshwargoswami1281** [Ayushsingh07/HELPERR#113] [Q&A] [answered]
libraries for implimentig facial recognition in js ??

**Comment by Ayushsingh07** [comment, marked as answer]
face-recognition.js 
u can go through this blog for more details 
https://itnext.io/face-api-js-javascript-api-for-face-recognition-in-the-browser-with-tensorflow-js-bcc2a6c4cf07

**Reply by omeshwargoswami1281** [reply]
thanks

**Comment by omeshwargoswami1281** [comment]
do i have to create frontend for this too??

**Reply by Ayushsingh07** [reply]
its up to u
